tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -583,7 +585,17 @@
             return;  // we just triggered that handler; don't need another
           }
         }
-        setString(TAB, selectedDashboard || '', /*useLocalStorage=*/false);
+
+        const pluginString = selectedDashboard || '';
+        setString(TAB, pluginString, /*useLocalStorage=*/false);
+
+        // Log the selection of this dashboard.
+        logEvent({
```

Comment:
You probably want something more like:

```js
ga('set', 'page', '/' + pluginString);
ga('send', 'pageview');
```

See https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications. Note the "Handling multiple URLs for the same resource" section which basically recommends we remove the hash.

Reply:
Yes! Indeed, that sounds like what we need. Navigating to a dashboard is basically a page view.